### PR TITLE
Increased sleep time to allow stork upgrade to complete

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:
Increased sleep time to allow stork upgrade to complete

**Which issue(s) this PR fixes** (optional)
Closes #PA-1298

**Special notes for your reviewer**:

### CODE REVIEW

**Review by snigdha-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by trenukarya-px — APPROVED** [review]

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/backup/backup_helper.go [hunk 1]
```diff
@@ -2555,7 +2555,7 @@ func upgradeStorkVersion(storkImageToUpgrade string) error {
 		}
 	}
 	// Sleep for upgrade request to go through before validating.
-	time.Sleep(10 * time.Second)
+	time.Sleep(5 * time.Minute)
```
Comment (lsrinivas-pure) [inline tests/backup/backup_helper.go:2558]: can we a check for upgrade  rather than sleep?

Comment (mkoppal-px) [reply tests/backup/backup_helper.go]: Done

**Comment by mkoppal-px** [comment]
[Jenkins run](https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/upgrade-tests/job/Stork-Upgrade/job/Stork-Upgrade-Vanilla/16/)

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**